packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/redeemable_payments_app_extension_schema.ts

```diff
@@ -0,0 +1,55 @@
+import {BaseSchema} from '../../schemas.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export type RedeemablePaymentsAppExtensionConfigType = zod.infer<typeof RedeemablePaymentsAppExtensionSchema>
+
+const MAX_LABEL_SIZE = 50
+
+export const RedeemablePaymentsAppExtensionSchema = BaseSchema.extend({
+  targeting: zod.array(zod.object({target: zod.literal('payments.redeemable.render')})).length(1),
```

Comment:
I think it'd make sense to extract `'payments.redeemable.render'` to a const here that can then be imported from the specification, so that you don't need to hardcode the target there as well